test/extended/util/framework.go

```diff
@@ -236,127 +236,157 @@ func processScanError(log string) error {
 // WaitForOpenShiftNamespaceImageStreams waits for the standard set of imagestreams to be imported
 func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
 	ctx := context.Background()
+	langs := []string{"ruby", "nodejs", "perl", "php", "python", "mysql", "postgresql", "jenkins"}
+	e2e.Logf("waiting for image ecoystem imagestreams to be imported")
+	for _, lang := range langs {
+		err := WaitForOpenShiftNamespaceImagestream(ctx, oc, lang)
+		if err != nil {
+			DumpSampleOperator(oc)
+			return err
+		}
+	}
+	return nil
+}
 
+// WaitForOpenShiftNamespaceImagestream waits for an individual imagestream in the openshift namespace to be imported.
+func WaitForOpenShiftNamespaceImagestream(ctx context.Context, oc *CLI, imagestream string) error {
```

Comment:
aside from the name change here @adambkaplan to `WaitForOpenShiftNamespaceImagestreamsManagedBySamplesOperator` let's make sure godoc is clear that sampels operator does not managae the imagestreams defined in https://github.com/openshift/cluster-samples-operator/blob/master/manifests/08-openshift-imagestreams.yaml and that this method and the ones it calls by no means confirms whether the imports on those in payload imagestreams has completed

i.e. the progressing / available / degraded conditions on samples operator are in no way affected by the import status of those in payload imagestreams